app/assets/javascripts/users.js

```diff
@@ -0,0 +1,23 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
can you wrap all of this in iife, so we don't pollute the global space?

https://developer.mozilla.org/en-US/docs/Glossary/IIFE

Reply:
I can look into that.